test/e2e/agent/config_test.go

```diff
@@ -150,10 +151,14 @@ func TestFleetMode(t *testing.T) {
 			WithDefaultESValidation(agent.HasWorkingDataStream(agent.LogsType, "elastic_agent.filebeat", "default")).
 			WithDefaultESValidation(agent.HasWorkingDataStream(agent.LogsType, "elastic_agent.metricbeat", "default")).
 			WithDefaultESValidation(agent.HasWorkingDataStream(agent.MetricsType, "elastic_agent.elastic_agent", "default")).
-			// to be re-enabled when https://github.com/elastic/cloud-on-k8s/issues/7389 is resolved
-			//WithDefaultESValidation(agent.HasWorkingDataStream(agent.MetricsType, "elastic_agent.filebeat", "default")).
 			WithDefaultESValidation(agent.HasWorkingDataStream(agent.MetricsType, "elastic_agent.metricbeat", "default"))
 
+		// https://github.com/elastic/cloud-on-k8s/issues/7389
+		if v.LT(version.MinFor(8, 12, 0)) && v.GE(version.MinFor(8, 14, 0)) {
```

Comment:
```suggestion
		if v.LT(version.MinFor(8, 12, 0)) || v.GE(version.MinFor(8, 14, 0)) {
```
I think you meant OR here, and the others as well.

Reply:
Yes! 🤦 
Thanks.